doc/configuration.rst

```diff
@@ -481,18 +481,24 @@ We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
   missing_data=20
   task_failed=30
   scheduling_error=35
+  unknown_reason=38 or 0
   unhandled_exception=40
 
 unhandled_exception
   For internal Luigi errors.  Defaults to 4, since this type of error
   probably will not recover over time.
+unknown_reason
+  For when a task does not run successfully because of an unknown reason. Despite
+  this case can be expected in an error free execution, it does not guarantee
+  completeness of the root task. Return code 38 is advised only for cases in which
+  Luigi return code 0 is used by the user to guarantee root tasks' completeness.
```

Comment:
You're right, lets not complicate things. The benefit I was looking for was to say that this return code is "no problem, don't worry".

What about using a number but a much lower number? I noticed we already say they are in increasing order of severity (I had forgotten this). Maybe use the number `15`?